composer.json

```diff
@@ -28,10 +28,10 @@
         "mockery/mockery": "~0.9"
     },
     "autoload": {
-        "psr-0": { "Humbug\\": "src/" }
+        "psr-4": { "Humbug\\": "src/Humbug" }
```

Comment:
It works without, but the whole change is about adding more beauty, so I cannot argue.